src/server/modes/charts/plugins/datalens/preparers/scatter/d3.ts

```diff
@@ -1,3 +1,4 @@
+import {SymbolType} from '@gravity-ui/chartkit/build/constants';
```

Comment:
Some kind of not very nice import path. I would like to see something like `@gravity-ui/chartkit/d3` here